qiling/os/windows/const.py

```diff
@@ -588,3 +587,37 @@
 # ...
 ThreadHideFromDebugger = 0x11
 # ...
+
+
+# https://docs.microsoft.com/en-us/windows/win32/api/winuser/nf-winuser-messageboxw
+
+MB_ABORTRETRYIGNORE = 0x000000020
+MB_CANCELTRYCONTINUE = 0x000000060
```

Comment:
does all these valiue works from 7 till 10 " maybe we need some additional comments if there are some version specific Windows

Reply:
I can be wrong, but i think every windows version support this